### DISCUSSION TITLE
Welcome to ModernUO Discussions!

### DISCUSSION THREAD

**Discussion by kamronbatman** [modernuo/ModernUO#730] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cavalier67** [comment]
How far along is your development?  
I'm just looking at creating my own shard.  Ran across a reference to you while researching ServUO. 
You should get the word out, sop people come find you.  The discussions area is a a ghost town

**Comment by kamronbatman** [comment]
Development is going well. So far there are only minor bugs reported. The main questions are on how to do certain things. Unfortunately the community is small, and help is hard to get, so building the documentation is not being worked on. There are a few shards released with ModernUO, and several shards in progress. Most discussing is on the [Discord](https://muo.gg/discord).
